redbot/cogs/customcom/customcom.py

```diff
@@ -292,6 +292,10 @@ async def cc_create_random(self, ctx: commands.Context, command: str.lower):
 
         Note: This command is interactive.
         """
+        if " " in command:
```

Comment:
Considering that d.py checks for any whitespace character when getting parts of the command, I wonder if we shouldn't be doing the same thing here too, for example:
```suggestion
        if any(char.isspace() for char in command):
```

d.py source: https://github.com/Rapptz/discord.py/blob/master/discord/ext/commands/view.py#L119

Reply:
Done